tests/Dockerfile.vic-integration-test

```diff
@@ -0,0 +1,15 @@
+# VIC Test Image
+#
+# Build a docker image that runs latest Ubuntu with golang, robotframework and vmware-vic installed
+# (https://golang.org/)
+# (http://robotframework.org/)
+# (https://github.com/vmware/vic)
+#
+# docker build -t vic-integration-test --no-cache -f Dockerfile.vic-integration-test .
+
+from vic-integration-test-base
+
+# Install latest version of vmware-vic
+run go get github.com/vmware/vic; echo 0
```

Comment:
I think this has less to do with PRs vs what to test.  With the `go get` you pull from the top of tree irrespective of what is checked out in *this* changset.  We should keep the testing framework in lock step with what's being tested.  It's possible the framework in this changeset will be some number of commits behind whatever MASTER is pointing to and will get wrong or inconsistent results.